Library/Formula/gtk+3.rb

```diff
@@ -43,16 +44,21 @@ def install
       --disable-glibtest
       --enable-introspection=yes
       --disable-schemas-compile
-      --enable-quartz-backend
-      --disable-x11-backend
     ]
 
+    if OS.linux?
```

Comment:
Please swap the sense of this test and use `if OS.mac?` instead since Quartz is specific to Mac.